Mathlib/Topology/Algebra/SeparationQuotient.lean

```diff
@@ -391,6 +409,14 @@ theorem mk_outCLM (x : SeparationQuotient E) : mk (outCLM K E x) = x :=
 @[simp]
 theorem mk_comp_outCLM : mk ∘ outCLM K E = id := funext (mk_outCLM K)
 
+variable {K} in
+theorem postcomp_mkCLM_surjective {L : Type*} [Semiring L] (σ : L →+* K)
```

Comment:
This lemma isn't a misisng instance; can you mention it in the PR description, or bump it to a later PR?

bors d+